.github/workflows/missing_types.yml

```diff
@@ -0,0 +1,29 @@
+name: Missing Types
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  create_issue:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'missing-types')
+    steps:
+      - name: Create an issue
+        uses: octokit/request-action@v2.x
```

Comment:
Can this just be a major version?

Reply:
Sorry, didn't got your question. Do you mean if it can use another version of the action?

Reply:
Sorry, I didn't really ask that well! Can it be a major tag i.e. `octokit/request-action@v2` as opposed to using both a major and minor (`v2.x`)?

Reply:
Gotcha.

Yes, it can be just a major, I'll replace it in next commit.